cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/RepositoryManager.java

```diff
@@ -338,6 +338,10 @@ public String cloneRemote()
     return resolveHead().getName();
   }
 
+  public String getCurrentCommit() throws IOException {
```

Comment:
I don't think we should have this method, as it encourages error prone behavior. It's better to return the commit id in the relevant push/pull methods.

Reply:
done